resources/views/partials/errors.blade.php

```diff
@@ -0,0 +1,9 @@
+@if ($errors->any())
+    <div class="alert alert-danger" role="alert">
+        <ul>
```

Comment:
Could you add `class="my-0"` to this `ul` to remove the uneven spacing? I know it's a mistake I left there, but while you're here.. ;)